ui/app/src/components/SystemStatus/GlobalDialogManager.tsx

```diff
@@ -429,6 +430,16 @@ function GlobalDialogManager() {
       />
       {/* endregion */}
 
+      {/* region Item Mega Menu */}
+      <ItemMegaMenu
+        open={state.itemMegaMenu.open}
+        path={state.itemMegaMenu.path}
+        onClose={createCallback(state.itemMegaMenu.onClose, dispatch)}
+        anchorReference={state.itemMegaMenu.anchorReference}
+        anchorPosition={state.itemMegaMenu.anchorPosition}
+      />
```

Comment:
Split `ItemMegaMenu` into container & presentational — right now it's sort of mixed. After the refactor, global dialog manager won't need to supply it with any props.